iothub/service/tests/DirectMethod/DirectMethodServiceRequestTests.cs

```diff
@@ -39,5 +39,41 @@ public void DirectMethodServiceRequest_Ctor_SetsMethodName()
             // assert
             dmcr.MethodName.Should().Be(expected);
         }
+
+        [TestMethod]
+        public void DirectMethodServiceRequest_ConnectionResponseTimeout()
+        {
+            // arrange
+            var expectedTimeout = TimeSpan.FromSeconds(1);
+            var dcmr = new DirectMethodServiceRequest("123")
```

Comment:
nit: Instead of having "123" as a method name, can we have something like "reboot" or "setTelemetryInterval"?